roles/nova-data/tasks/libvirt.yml

```diff
@@ -14,7 +14,7 @@
     - qemu-kvm={{ nova.qemu_kvm_version }}
     - "{{ nova.qemu_system_package }}={{ nova.qemu_kvm_version }}"
     - qemu-system-common={{ nova.qemu_kvm_version }}
-    - librbd1={{ nova.librdb1_version }}
+    - librbd1={{ ceph.client_version[os] }}
```

Comment:
this can break production env that have version override in all.yml. instead of changing it here update nova-data/defaults/main.yml --> nova.librdb1_version  variable value to {{ ceph.client_version[os] }}